lib/interactor/organizer.rb

```diff
@@ -76,7 +76,8 @@ module InstanceMethods
       # Returns nothing.
       def call
         self.class.organized.each do |interactor|
-          interactor.call!(context)
+          context.signal_early_return! if context.failure?
```

Comment:
I've extracted throwing symbol from `Context#fail` to a dedicated method so we can avoid logic duplication.